code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -147,3 +148,163 @@
+	user.visible_message(span_warning("Someone in [src] begins to wriggle!"), \
+		span_notice("You start wriggling, attempting to loosen [src]'s buckles... (this will take about [DisplayTimeText(breakout_time)].)"), \
+		span_hear("You hear straining cloth from [src]."))
+	if(do_after(user,(breakout_time), target = src))
+		if(!user || user.stat != CONSCIOUS || user.loc != src || opened || !sinched )
+			return
+		//we check after a while whether there is a point of resisting anymore and whether the user is capable of resisting
+		user.visible_message(span_danger("[user] successfully broke out of [src]!"),
+							span_notice("You successfully break out of [src]!"))
+		bust_open()
+	else
+		if(user.loc == src) //so we don't get the message if we resisted multiple times and succeeded.
+			to_chat(user, span_warning("You fail to break out of [src]!"))
+
+
+/obj/structure/closet/body_bag/environmental/prisoner/bust_open()
+	SIGNAL_HANDLER
+	sinched = FALSE
+	// We don't break the bag, because the buckles were backed out as opposed to fully broken.
+	open()
+
+/obj/structure/closet/body_bag/environmental/prisoner/RightClick(mob/user, modifiers)
+	if(!user.canUseTopic(src, BE_CLOSE) || !isturf(loc))
+		return
+	if(!opened)
+		togglelock(user)
+	return TRUE
+
+/obj/structure/closet/body_bag/environmental/prisoner/togglelock(mob/living/user, silent)
+	if(iscarbon(user))
+		add_fingerprint(user)
+	if(!sinched)
+		for(var/mob/living/target in src.contents)
+			to_chat(target, span_userdanger("You feel the lining of [src] tighten around you! Soon, you won't be able to escape!"))
+		user.visible_message(span_notice("You begin sinching down the buckles on [src]."))
+		if(!(do_after(user,(sinch_time),target = src)))
+			return
+	sinched = !sinched
+	user.visible_message(span_notice("[user] [sinched ? null : "un"]sinches [src]"),
+							span_notice("You [sinched ? null : "un"]sinch [src]"),
+							span_hear("You hear stretching followed by metal clicking from [src]."))
+	log_game("[key_name(user)] [sinched ? "sinched":"unsinched"] secure environmental bag [src] at [AREACOORD(src)]")
+	update_appearance()
+
+/obj/structure/closet/body_bag/environmental/prisoner/syndicate
+	name = "syndicate prisoner transport bag"
+	desc = "An alteration of Nanotrasen's environmental protection bag which has been used in several high-profile kidnappings. Designed to keep a victim unconscious, alive, and secured during transport."
+	icon = 'icons/obj/bodybag.dmi'
+	icon_state = "syndieenvirobag"
+	pressure_protection = 1
+	thermal_insulation = 1
+	foldedbag_path = /obj/item/body_bag/environmental/prisoner/syndicate
+	weather_protection = list("all")
+	breakout_time = 4800
+	sinch_time = 300
+	var/obj/item/tank/internals/anesthetic/tank // todo: make this transfer over to the held item, currently the tank is bottomless by folding and unfolding the bag.
+
+/obj/structure/closet/body_bag/environmental/prisoner/syndicate/Initialize()
+	. = ..()
+	tank = new
```

Comment:
this is not deleted anywhere... i'm pretty sure this is either a hard delete or something that is just gonna never get cleaned up